strr-base-web/app/composables/useStrrBasePermit.ts

```diff
@@ -44,19 +44,10 @@ export const useStrrBasePermit = <R extends ApiRegistrationResp, A extends ApiAp
       application.value = await updatePaymentDetails<A>(application.value.header.applicationNumber)
     }
 
-    // check if current application is an approved Platform, for which registration would need to be loaded
-    const isApprovedPlatform =
-      application.value?.header.registrationStatus === RegistrationStatus.ACTIVE &&
-        application.value?.registration?.registrationType === ApplicationType.PLATFORM
-
-    const isRenewalDraftPlatform =
-      application.value?.header.registrationStatus === RegistrationStatus.EXPIRED &&
-        application.value?.header.status === ApplicationStatus.DRAFT &&
-        application.value?.registration?.registrationType === ApplicationType.PLATFORM
-
     // conditions when registration needs to be loaded
     const shouldLoadRegistration =
-      application.value?.header.applicationType !== 'renewal' || isApprovedPlatform || isRenewalDraftPlatform
+      application.value?.header.applicationType !== 'renewal' ||
+        application.value?.registration?.registrationType === ApplicationType.PLATFORM
```

Comment:
Before it loaded for specific statuses (ACTIVE or EXPIRED and DRAFT) is that no longer applicable now?

Reply:
I think for Platforms we need to load registrations regardless of status. So i simplified the logic. Thoughts?